geometry_msgs/Point[] waypoints
---
float64 cost
